src/commissaire/models/__init__.py

```diff
@@ -76,8 +76,9 @@ def __init__(self, **kwargs):
         for key in self._attribute_map.keys():
             if key not in kwargs:
                 raise TypeError(
-                    '__init__() missing 1 or more required '
-                    'keyword arguments: {0}'.format(
+                    '{0}.__init__() missing 1 or more required '
```

Comment:
nit: We can now drop the numbers in the templates since we've moved to Python 3. No need to change it now, but when there isn't a need to number them we can simply move to ordered '{}'.